src/config.js

```diff
@@ -0,0 +1,7 @@
+const config = {
```

Comment:
Have you considered importing configs from dojot-module-nodejs ?

Reply:
Nice! I'll do that!

Reply:
Hey Matheus! Thank you for commenting. But, I was implementing dojot-module-nodejs, and I have an impression, the lib is too large for this application. With lib, we will have Kafka, for example, consumer and producer, and this is not necessary.